### ISSUE TITLE
<picture> element should allow aria-hidden

### ISSUE THREAD

**Issue by seasonedfish** [jsx-eslint/eslint-plugin-jsx-a11y#1095]
A `<picture>` element is allowed to have aria-hidden since October 2021, per the [W3C Recommendation](https://www.w3.org/TR/html-aria/#:~:text=26%20October%202021%20%2D%20Addition%3A%20Allow%20aria%2Dhidden%20attribute%20on%20the%20picture%20element.).

```html
<picture aria-hidden="true"></picture>
```

The following lint is emitted currently:
```
This element does not support ARIA roles, states, or properties.
help: Try removing the prop `aria-hidden`.
StorytellingSection.tsx(169, 20): 
```
